src/styles/index.scss

```
@@ -1,3 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  &__img {
+    width: 100%;
+    height: 134px;
+    padding: 0 3px;
+  }
+  &__list {
+    display: flex;
+    flex-direction: column;
+    .name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: #060b35;
+      margin: 36px 0 4px;
+    }
+    .code {
```

Add the blank line before the next selector, fix it everywhere
```suggestion
    }
    
    .code {
```